File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.py
def solution(myString, pat):
    answer = 0
    
    myString = list(myString)
    for i in range(len(myString)):
        if myString[i] == 'A':
            myString[i] = 'B'
        else:
            myString[i] = 'A'
    myString = ''.join(myString)    
    return 1 if pat in myString else 0

File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.js
function solution(myString, pat) {
    var answer = 0;
    
    myString = myString.replaceAll('A', 'C').replaceAll('B','A').replaceAll('C','B')
    return  myString.indexOf(pat) !== -1 ? 1 : 0
    
}